_includes/selector.html

```diff
@@ -389,9 +377,9 @@
             img_loc: ["NGC", "Dockerhub"],
             img_types: ["Base", "Notebooks"],
             packages: ["Standard", "Choose Specific Packages"],
-            additional_pip_packages: ["cuDF", "dask-cuDF", "cuML", "cuGraph", "cuXFilter", "cuSpatial", "cuCIM"],
+            additional_pip_packages: ["cuDF", "dask-cuDF", "cuML", "cuGraph", "cuSpatial", "cuXFilter", "cuCIM"],
```

Comment:
I just noticed that we're inconsistent about the casing of `cuxfilter`, we have `cuXfilter` and `cuXFilter` written in this page. The README and docs say `cuxfilter` with no capital letters. @exactlyallan Can you verify if the name should be written `cuxfilter`?

Reply:
I like `cuxfilter`.

Reply:
Lower-cased all instances in https://github.com/rapidsai/docs/pull/414/commits/6323860c0192398c60bbd9b5381def1d1290112d